modules/core/src/main/scala/org/scalasteward/core/edit/update/Selector.scala

```diff
@@ -84,6 +88,12 @@ object Selector {
   private def scalaCliUsingLib: Pattern =
     Pattern.compile("""//>\s+using\s+lib\s+""")
 
+  private def scalaCliUsingDep: Pattern =
+    Pattern.compile("""//>\s+using\s+dep\s+""")
+
+  private def scalaCliUsingTestDep: Pattern =
+    Pattern.compile("""//>\s+using\s+test.dep\s+""")
```

Comment:
does it make sense to have these 3 pattern as a `val` instead of re creating them over and over again ?

Reply:
Absolutely 👍 Thanks for spotting that!